doc/base/classes.xml

```diff
@@ -20058,13 +20060,41 @@
 			</return>
 			<description>
 				Return the text in the [LineEdit].
+        
+        [code]
```

Comment:
You can't use `[code]` for that, it's meant for inline code. You can however use `[codeblock]`.